specs/protocol/tx_format.md

```diff
@@ -294,15 +323,23 @@ The balance root `balanceRoot` is the root of the [SMT](./cryptographic_primitiv
 
 The state root `stateRoot` is the root of the [SMT](./cryptographic_primitives.md#sparse-merkle-tree) of storage slots. Each storage slot is a `byte[32]`, keyed by a `byte[32]`.
 
-### OutputWithdrawal
+### OutputMessage
+
+| name                  | type        | description                                                             |
+|-----------------------|-------------|-------------------------------------------------------------------------|
+| `messageID`           | `byte[32]`  | The messageID as described [here](./identifiers.md#message-id).         |
```

Comment:
Wait, why does this have a message ID? Isn't that only for inputs?

Reply:
Right now, the message ID for outputs is what's placed in the binary merkle tree that gets committed to Ethereum. It's these message IDs that the Inbox uses to ensure a message is never relayed more than once

Reply:
Ah I see. In that case, is there really a need for the rest of the fields? Those can be placed in the receipt, rather than in the output.

Reply:
I thought we weren't doing receipts for messages because all the data is already in the output? The fields needs to be somewhere that's easy for a user to pull from a client so anyone can then relay the messages on L1

Reply:
I've now moved most of the OutputMessage data into a receipt. Data length is capped at 2**16 or 64kB